### PR TITLE
feat(RickBot): add presence

### PR DESCRIPTION
**Pull request by ghost**
I added a new rich presence for RickBot.

### CODE REVIEW

**Review by EncryptedDev — COMMENTED** [review]
ignore

File: websites/R/RickBot/presence.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+presence.on("UpdateData", async () => {
+  // Presence
+  const presenceData: PresenceData = {
+    largeImageKey: "rickastley",
+    startTimestamp: timeElapsed
+  };
+
+  // Default State
+  if (document.location.hostname === "rickbot.net")
```
Comment (Slowlife01) [inline websites/R/RickBot/presence.ts:57]:
```suggestion
  if (document.location.pathname === "/")
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Show proof of the presence working afterwards

**Comment by Slowlife01** [comment]
Inactive

**Pull request closed without merging**